scripts/generate_tf_guides.py

```diff
@@ -164,6 +163,13 @@ def generate_single_tf_guide(source_dir, target_dir, title, source_name, target_
             if len(lines) < 2:
                 new_lines.append(lines[-1])
             cell["source"] = new_lines
+        elif cell["cell_type"] == "code":
+          lines = cell["source"]
+          if lines[0] in ('\n', ' ', ''):
```

Comment:
Prefer using `if not lines[0].strip()`, it should be more general

Reply:
Done